jenkins/scripts/slurm_run.sh

```
@@ -79,31 +79,45 @@ pytest_exit_code=0
+    --group "${testGroup:-1}"
+    --pytest-base-cmd "$pytestCommand"
+    --stage-name "$stageName"
+    --output-dir "$jobWorkspace"
+    --working-dir "$llmSrcNode/tests/integration/defs"
+    --fail-signatures "${failSignaturesList:-}"
+    --max-rerun-tests 5
+)
+if [ -n "${testDurationsPath:-}" ]; then
+    runTestsArgs+=(--durations-path "$testDurationsPath")
+fi
+if [ "$perfMode" = "true" ]; then
```

Shellcheck (0.11.0)

[warning] perfMode is referenced but not assigned.

(SC2154)